src/instruments/src/EFB/Assets/Efb.scss

```diff
@@ -116,6 +116,20 @@ html body {
 
 .leaflet-container {
   @apply rounded-2xl;
+
+  .leaflet-control-attribution {
+    display: none;
+  }
+}
+
+.credits {
+  font-family: 'IBMPlexMono';
```

Comment:
we can extend this to be used as default mono in tailwind config.

Reply:
I think I already did it default mono in tailwind config, if I'm not mistaken. 😎

Reply:
Yes. It was. `fontFamily: { mono: ['IBMPlexMono'], efb: ['NunitoSans'] },`

Reply:
Cool, we can then use `@apply font-mono;` here.